### PR TITLE
DAH-2510 listing details page status components

### PR DESCRIPTION
**Pull request by tallulahkay**
DAH-2510

### CODE REVIEW

File: app/javascript/modules/listingDetailsAside/ListingDetailsApplicationDate.tsx [hunk 1]
```diff
@@ -1,35 +1,55 @@
...
+        }
+      >
+        <div>
+          <span>{getMessage(isApplicationClosed, isFcfs)}</span>
+          {isFcfs ||
+            (!isApplicationNotYetOpen && (
+              <>
+                {": "}
+                <span className="font-bold">
+                  {localizedFormat(listing.Application_Due_Date, "LL")}
```
Comment (tallulahkay) [inline app/javascript/modules/listingDetailsAside/ListingDetailsApplicationDate.tsx:40]: Is the application due date the correct field for when fcfs applications open?

**Review by jimlin-sfgov — APPROVED** [review]
LGTM

File: app/assets/json/translations/locale-es.json [hunk 2]
```diff
@@ -470,7 +470,6 @@
                 "lottery_will_be_held": "El sorteo se llevará a cabo el %{lottery_date}. Los resultados del sorteo de viviendas se publicarán en %{link}.",
                 "office_hours": "Horarios de atención",
                 "on_the_listing": "en el listado",
-                "notify_about_potential_resend": "<b>Si ya recibió este correo electrónico de confirmación de nuestra parte, ignore este mensaje.</b> Tuvimos un problema al enviar correos electrónicos desde DAHLIA recientemente, por lo que los enviamos nuevamente. Hicimos esto para asegurarnos de que todos los solicitantes tengan sus números de lotería.",
```
Comment (jimlin-sfgov) [inline app/assets/json/translations/locale-es.json:473]: @tallulahkay Something is wrong with our Angular translations in phrase, probably related to some work I did for the spam incident. I'm looking into this now.

Comment (jimlin-sfgov) [reply app/assets/json/translations/locale-es.json:473]: @tallulahkay I've updated Angular translations in phrase. Running `grunt phrasePull` again should fix this issue.

**Pull request merged**